Parser/OperatingSystem.php

```diff
@@ -361,13 +344,146 @@ public static function getNameFromId(string $os, ?string $ver = null): ?string
         return null;
     }
 
+    /**
+     * Returns the OS that can be safely detected from client hints
+     *
+     * @return array
+     */
+    protected function parseOsFromClientHints(): array
+    {
+        $name = $version = $short = '';
+
+        if ($this->clientHints instanceof ClientHints && $this->clientHints->getOperatingSystem()) {
+            $hintName = $this->clientHints->getOperatingSystem();
+
+            if ('macOS' === $hintName) {
+                $hintName = 'Mac';
+            } elseif ('Linux' === $hintName) {
+                $hintName = 'GNU/Linux';
+            }
```

Comment:
maybe we should make an array?
```php
$hintOsNames = ['macOS' => 'Mac', 'Linux' => 'GNU/Linux' ];
$hintName = $hintOsNames[$hintName] ?? $hintName;
```
can also make all keys in lowercase

Reply:
Was also thinking of some kind of static mapping array. Same for browsers... Guess over time we will need to add more entries there, as soon as more browsers start supporting it